code/game/dna/dna2.dm

```diff
@@ -154,9 +162,12 @@ var/global/list/bad_blocks[0]
 	var/bodyacc	= character.body_accessory*/
 
 	// Markings
-	if(!character.m_style)
-		character.m_style = "None"
-	var/marks	= marking_styles_list.Find(character.m_style)
+	if(!character.m_styles)
+		character.m_styles = initial(character.m_styles)
```

Comment:
Due to how BYOND initializes lists, initial doesn't work with them. You'll probably want to put their initial value in a `#define` and use that anywhere you initialize it.